x-pack/legacy/plugins/siem/public/pages/network/ip_details.tsx

```diff
@@ -49,234 +47,227 @@ const TlsTableManage = manageQuery(TlsTable);
 const UsersTableManage = manageQuery(UsersTable);
 const IpOverviewManage = manageQuery(IpOverview);
 
-interface IPDetailsComponentReduxProps {
-  filterQuery: string;
-  flowTarget: FlowTarget;
-  setAbsoluteRangeDatePicker: ActionCreator<{
-    id: InputsModelId;
-    from: number;
-    to: number;
-  }>;
-}
-
-export type IPDetailsComponentProps = IPDetailsComponentReduxProps & { detailName: string };
-
 export const IPDetailsComponent = pure<IPDetailsComponentProps>(
-  ({ detailName, filterQuery, flowTarget, setAbsoluteRangeDatePicker }) => (
+  ({
+    detailName,
+    filterQuery,
+    flowTarget,
+    setAbsoluteRangeDatePicker,
+    to,
+    from,
+    setQuery,
+    isInitializing,
+  }) => (
     <>
       <WithSource sourceId="default" data-test-subj="ip-details-page">
         {({ indicesExist, indexPattern }) => {
           const ip = decodeIpv6(detailName);
           return indicesExistOrDataTemporarilyUnavailable(indicesExist) ? (
             <StickyContainer>
-              <GlobalTime>
```

Comment:
moved `GlobalTime` here https://github.com/elastic/kibana/pull/47556/files#diff-837a0ad8db92736e389aba8d96e8d13cR27 to match structure from `hosts`